module/platform/Webots/src/Webots.cpp

```diff
@@ -260,18 +270,27 @@ void Webots::translate_and_emit_sensor(const SensorMeasurements& sensor_measurem
     for (const auto& position : sensor_measurements.position_sensors) {
         // string name
         // double value
+        translate_servo_id(position.name, sensor_data.servo).presentPosition = position.value;
     }
 
     for (const auto& accelerometer : sensor_measurements.accelerometers) {
         // string name
         // Vector3 value
+        sensor_data.accelerometer.x = static_cast<float>(accelerometers.value.X);
+        sensor_data.accelerometer.y = static_cast<float>(accelerometers.value.Y);
+        sensor_data.accelerometer.z = static_cast<float>(accelerometers.value.Z);
```

Comment:
A couple of issues with this I think. I don't think `accelerometers` is defined (the right side variable which owns the values). I think that was supposed to be `accelerometer` - the thing the for loop gives you.

The other thing is I am pretty sure that `sensor_data.accelerometers` (note the `s`) is an array, where you actually want to be assigning to the specific id associated with that servo. This will probably get messy.